docs/quick-start.md

```diff
@@ -30,12 +30,12 @@ kind: KadaluStorage
 metadata:
     # This will be used as name of PV Hosting Volume
     name: storage-pool-1
-    spec:
-        type: Replica1
-        storage:
-        - node: kube1      # node name as shown in `kubectl get nodes`
-          device: /dev/vdc # Device to provide storage to all PVs
-```
+spec:
+    type: Replica1
+    storage:
+      - node: minikube      # node name as shown in `kubectl get nodes`
```

Comment:
change back to kube1, and actual device as earlier. Minikube example is not prefered for the website